src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -12,16 +12,21 @@ export const ClusterPlatformIntegrationHint: React.FC<ClusterPlatformIntegration
   platformType,
 }) => {
   const { isPlatformIntegrationSupported } = useClusterSupportedPlatforms(clusterId);
-  if (isPlatformIntegrationSupported && platformType !== 'vsphere') {
+  if (isPlatformIntegrationSupported) {
```

Comment:
The condition should be `!hasActivePlatformIntegration`, which is either platformType is any of the items in 

`const platformsSupported = ['vpshere', 'nutanix'];`

This constant should be exposed in a shared file, so we can refer to the supported platforms from different places.